src/raftstore/store/peer_storage.rs

```diff
@@ -75,6 +75,35 @@ impl PartialEq for SnapState {
     }
 }
 
+// Discard all log entries prior to compact_index. We must guarantee
+// that the compact_index is not greater than applied index.
+pub fn compact_raft_log(tag: &str,
```

Comment:
The name compact_raft_log confuses me, it just only updates the truncated info to the state, not really compacts raft log.